app/views/shared/_user.html.erb

```diff
@@ -1,14 +0,0 @@
-
-<div class="card mb-3" style="max-width: 540px;">
-  <div class="row g-0">
-    <div class="col-md-4">
-      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQGV-z4WJHz0Ci9sdY8m4DAbUmgRB0VskykuGOXmpPKBg&s" class="img-fluid rounded-start" alt="...">
-    </div>
-    <div class="col-md-8">
-      <div class="card-body">
-        <h5 class="card-title">User Name</h5>
-      <p class="card-text"><small class="text-muted">Numbers of posts</small></p>
```

Comment:
- [**Optional**] Great job for making use of partials :+1:  To make sure our design matches the wireframe, let's apply the changes in the screenshot below: :smile: 

![annotely_image (12)](https://user-images.githubusercontent.com/63087491/191869324-5f57a90d-e879-4893-850f-bb7e5dbb13b6.png)